client/signup/steps/rewind-were-backing/index.jsx

```diff
@@ -25,21 +26,27 @@ class RewindWereBacking extends Component {
 
 		// Connected props
 		siteSlug: PropTypes.string.isRequired,
+		dependencyStore: PropTypes.object,
 	};
 
 	stepContent = () => {
-		const { translate, siteSlug } = this.props;
+		const { translate, siteSlug, dependencyStore } = this.props;
 
 		return (
 			<Card className="rewind-were-backing__card rewind-switch__card rewind-switch__content">
 				<h3 className="rewind-were-backing__title rewind-switch__heading">
-					{ translate( "We're backing up your site!" ) }
+					{ translate( 'Your site is set up and ready!' ) }
 				</h3>
 				<img src="/calypso/images/illustrations/thankYou.svg" alt="" />
 				<p className="rewind-were-backing__description rewind-switch__description">
+					{ get( dependencyStore, 'rewindconfig', false ) &&
```

Comment:
This does work, but it seems like we should be checking the SM here, rather than relying on the dependency store.

Reply:
Makes sense 👍
I'll wait until Michael responds about exiting the flow in the second skip.